setup.py

```diff
@@ -38,7 +38,7 @@ def read_requirements(filename: str) -> List[str]:
         "Programming Language :: Python :: 3.10",
         "Topic :: Software Development :: Libraries",
     ],
-    package_dir={"": "fideslib"},
+    packages=find_packages(exclude=("tests*",)),
```

Comment:
In this case I don't think we do since this doesn't have a `src` directory. Since `fideslib` is the root module directory I don't think we need it. Admittedly I normally use poetry which handles this stuff for you so I wasn't sure. I looked up some other projects I know and this is how they did it with no `src` directory.

Reply:
alright thanks for explaining